python-sdk/tests/benchmark/dags/evaluate_load_file.py

```diff
@@ -73,9 +75,74 @@ def create_dag(database_name, table_args, dataset, kwargs):
             "output_table": table,
             "chunk_size": chunk_size,
         }
-        # Override params from config
-        params.update(kwargs)
 
+        # Override params from config
+        # example config:
+        # {
+        #     "databases": [
+        #         {
+        #             "kwargs": {                            # global config bigquery db
+        #                 "enable_native_fallback": false
+        #             },
+        #             "name": "biquery",
+        #             "output_table": {
+        #                 "conn_id": "biquery"
+        #             }
+        #         },{
+        #              "kwargs": {                          # global config for redshift db
+        #                   "enable_native_fallback": false
+        #               },
+        #               "name": "redshift",
+        #               "output_table": {
+        #                   "conn_id": "redshift"
+        #               }
+        #         }],
+        #       "datasets": [
+        #             {
+        #                 "conn_id": "aws",
+        #                 "file_type": "parquet",
+        #                 "name": "ten_kb",
+        #                 "path": "s3://astro-sdk/benchmark/trimmed/covid_overview/covid_overview_10kb.csv",
+        #                 "rows": 160,
+        #                 "size": "10 KB",
+        #                 "database_kwargs" : {
+        #                      "biquery": {                           # local config for bigquery db
+        #                          native_support_kwargs: {"skip_leading_rows": 1}
+        #                      },
+        #                      "redshift": {                          # local config for redshift db
+        #                            native_support_kwargs:{
+        #                               "IGNOREHEADER": 1,
+        #                               "REGION": "us-west-2",
+        #                               "IAM_ROLE": "REDSHIFT_NATIVE_LOAD_IAM_ROLE_ARN",
+        #                             }
+        #                      }
+        #                  }
+        #             }]
+        # }
+        #
+        # Final config for load_file()
+        #     1. Redshift
+        #         {
+        #             native_support_kwargs: {
+        #                 "IGNOREHEADER": 1,
+        #                 "REGION": "us-west-2",
+        #                 "IAM_ROLE": REDSHIFT_NATIVE_LOAD_IAM_ROLE_ARN".
+        #             },
+        #             "enable_native_fallback": false
+        #         }
+        #     2.  Bigquery
+        #         {
+        #             native_support_kwargs: {
+        #                 "skip_leading_rows": 1,
+        #             },
+        #             "enable_native_fallback": false
+        #         }
+        # Note - If there is a config present in both local and global config.
+        # Local config will override the global one.
```

Comment:
shall we keep this example in config.json or docstring, WDYT?

Reply:
Moved it to doc string.